Bottom: e8d2d36735f59221b6ff5ff543baeaaa65b3dd7b
Top:    21a2c56815f249c90d5f694c002e803d08955c5a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:57 -0500

Use ansible docker_image module


---
diff --git a/roles/docker-build-base-image/tasks/main.yml b/roles/docker-build-base-image/tasks/main.yml
index b984d740..11c50d32 100644
--- a/roles/docker-build-base-image/tasks/main.yml
+++ b/roles/docker-build-base-image/tasks/main.yml
@@ -16,8 +16,8 @@
   with_dict: "{{ docker_build_base_images }}"
 
 - name: build image
-  docker_build:
-    image_name: '{{ item.value.image_name }}'
-    image_tag: '{{ item.value.image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/base-images/{{ item.key }}/{{ item.value.relative_dir }}'
+  docker_image:
+    name: '{{ item.value.image_name }}'
+    tag: '{{ item.value.image_tag }}'
+    path: '{{ docker_projects_dir }}/base-images/{{ item.key }}/{{ item.value.relative_dir }}'
   with_dict: "{{ docker_build_base_images }}"
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index ddd2a45e..43e565d4 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -83,10 +83,10 @@
     dest: '{{ docker_projects_dir }}/docker-exim4'
 
 - name: build image
-  docker_build:
-    image_name: '{{ exim4_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-exim4'
+  docker_image:
+    name: '{{ exim4_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-exim4'
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-gitlab/tasks/main.yml b/roles/docker-gitlab/tasks/main.yml
index e7aa983b..7e8556c1 100644
--- a/roles/docker-gitlab/tasks/main.yml
+++ b/roles/docker-gitlab/tasks/main.yml
@@ -81,14 +81,14 @@
     dest: '{{ docker_projects_dir }}/docker-gitlab'
 
 - name: build image
-  docker_build:
-    image_name: '{{ gitlab_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-gitlab'
+  docker_image:
+    name: '{{ gitlab_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-gitlab'
 
 # Data volume image is now the same as the container image
 #- name: build image
-#  docker_build:
+#  docker_image:
 #    image_name: '{{ gitlab_dv_image_name }}'
 #    image_tag: '{{ docker_image_tag }}'
 #    dockerfile_dir: '{{ docker_projects_dir }}/docker-gitlab'
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index dd57984f..d982b4e8 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -77,10 +77,10 @@
     dest: '{{ docker_projects_dir }}/docker-mantisbt'
 
 - name: build image
-  docker_build:
-    image_name: '{{ mantisbt_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-mantisbt'
+  docker_image:
+    name: '{{ mantisbt_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-mantisbt'
 
 - name: Checkout db repo
   git:
@@ -89,10 +89,10 @@
     dest: '{{ docker_projects_dir }}/docker-mysql-data'
 
 - name: build data-container image
-  docker_build:
-    image_name: '{{ mantisbt_db_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-mysql-data'
+  docker_image:
+    name: '{{ mantisbt_db_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-mysql-data'
 
 # *****************************************************************************
 # copy the MySQL config to the restore directory
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index ce0edd94..00e4c2b7 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -78,10 +78,10 @@
     dest: '{{ docker_projects_dir }}/docker-mediawiki'
 
 - name: build image
-  docker_build:
-    image_name: '{{ wiki_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-mediawiki'
+  docker_image:
+    name: '{{ wiki_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-mediawiki'
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index 4da8f5e6..c8c2170b 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -68,10 +68,10 @@
 
 
 - name: build image
-  docker_build:
-    image_name: '{{ openldap_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-openldap'
+  docker_image:
+    name: '{{ openldap_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-openldap'
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-openssl/tasks/main.yml b/roles/docker-openssl/tasks/main.yml
index 7d4ff5a7..7bb3bf84 100644
--- a/roles/docker-openssl/tasks/main.yml
+++ b/roles/docker-openssl/tasks/main.yml
@@ -61,10 +61,10 @@
     dest: '{{ docker_projects_dir }}/docker-openssl'
 
 - name: build image
-  docker_build:
-    image_name: '{{ openssl_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-openssl'
+  docker_image:
+    name: '{{ openssl_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-openssl'
 
 # *****************************************************************************
 # Create the data volumes
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index d1eeb954..4ca233a6 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -60,10 +60,11 @@
     dest: '{{ docker_projects_dir }}/docker-svn'
 
 - name: build image
-  docker_build:
-    image_name: '{{ svn_image_name }}'
-    image_tag: '{{ docker_image_tag }}'
-    dockerfile_dir: '{{ docker_projects_dir }}/docker-svn'
+  docker_image:
+    force: true
+    name: '{{ svn_image_name }}'
+    tag: '{{ docker_image_tag }}'
+    path: '{{ docker_projects_dir }}/docker-svn'
 
 # *****************************************************************************
 # Create the data volumes
